q2studio/server.py

```diff
@@ -13,14 +13,15 @@
 from flask import Flask
 from gevent.pywsgi import WSGIServer
 
-from q2studio.api import jobs, plugins, types, workspace
+from q2studio.api import jobs, plugins, types, formats, workspace
 from q2studio.security import validate_request_authentication
 from q2studio.headers import add_cors_headers
 
 studio = Flask('q2studio')
 studio.register_blueprint(jobs, url_prefix='/api/jobs')
 studio.register_blueprint(plugins, url_prefix='/api/plugins')
 studio.register_blueprint(types, url_prefix='/api/types')
+studio.register_blueprint(formats, url_prefix='/api/formats')
```

Comment:
(Sorry ^^ comment was on the wrong thread.  But glad you thumbs-up my registration of the blueprint :) )